http/src/lib.nr

```diff
@@ -75,67 +72,44 @@ impl Parser {
                 state.parsing_header_name = true;
                 state.parsing_start = 0;
                 state.line_status = 0;
-                state.line_digest = state.line_monomial * character as Field;
-                state.line_monomial *= polynomial_input;
             }
             // Handle parsing body
             else if state.line_status == 4 {
                 state.parsing_header = 0; // TODO: this is a little bit unintuitive, but we'll start counting headers at 1?
                 state.parsing_header_value = false;
                 state.parsing_body = true;
                 state.line_status = 0;
-                state.line_digest += state.line_monomial * character as Field;
```

Comment:
These operations are computing a running polynomial hash

Reply:
yes. I moved this logic to [here](https://github.com/pluto/noir-web-prover-circuits/pull/7/files#diff-a8f75d1a03eada28b71fa463b28478940a65d8da3df3b9c94244b9413c6fac82R239)